### PR TITLE
OPL-7: Include InvoiceLineItem services. 

### PR DESCRIPTION
**Pull request by dborowiecki**
TICKET: [OPL-7](https://openimis.atlassian.net/browse/OPL-7)

### CODE REVIEW

File: invoice_payment/services/invoiceLineItem.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        if isinstance(tax_data, str):
+            tax_data = json.loads(tax_data)
+        if 'total' not in tax_data.keys():
+            logger.warning("Invalid 'tax_analysis' format, total field not found")
+            return 0
+        else:
+            return float(tax_data['total'])
+
+
+
```
Comment (sniedzielski) [inline invoice_payment/services/invoiceLineItem.py:80]: is this large empty space made on purpose?

File: invoice_payment/services/base.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    def _adjust_create_payload(self, payload_data):
+        return self._base_payload_adjust(payload_data)
+
+    def _adjust_update_payload(self, payload_data):
+        return self._base_payload_adjust(payload_data)
+
+    def _base_payload_adjust(self, obj_data):
+        return obj_data
+
+    def _build_delete_instance_payload(self, obj_):
```
Comment (sniedzielski) [inline invoice_payment/services/base.py:77]: What do you think about move this to 'service utils'? I see in 'utils' are some functions to output some data etc.

**Review by sniedzielski — COMMENTED** [review]
@dborowiecki Looks really good, please refer to those two points, thx!

File: invoice_payment/services/invoiceLineItem.py (hunk 1 above)
Comment (dborowiecki) [reply invoice_payment/services/invoiceLineItem.py]: No, removed.

File: invoice_payment/services/base.py (hunk 2 above)
Comment (dborowiecki) [reply invoice_payment/services/base.py]: Good idea. It'll be more consistent to keep itwith other payload related methods.

**Review by delcroip — APPROVED** [review]

**Pull request merged**